tests/transforms/test_batch_input.py

```diff
@@ -24,11 +24,11 @@ def test_simple_circuit():
     """Test that batching works for a simple circuit"""
     dev = qml.device("default.qubit", wires=2)
 
-    @qml.batch_input(argnum=0)
+    @qml.batch_input(argnum=1)
```

Comment:
Very good to test the newly covered scenario! Would be great to also include the new test case that involves a parameter-free operation before the batched operation :)

Reply:
Added test 👍